config/jms/activemq-jms-config.yaml

```diff
@@ -0,0 +1,225 @@
+## KIE Servers BEGIN
+servers:
+## RANGE BEGINS
+#[[ range $index, $Map := .Servers ]]
+## KIE server deployment config BEGIN
+  - deploymentConfigs:
+    - metadata:
+        name: "[[.KieName]]"
+      spec:
+        template:
+          spec:
+            containers:
+              - name: "[[.KieName]]"
+                env:
+                  # JMS config BEGIN
+                  - name: KIE_SERVER_JMS_QUEUE_REQUEST
+                    value: "[[.Jms.KieServerJmsQueueRequest]]"
+                  - name: KIE_SERVER_JMS_QUEUE_RESPONSE
+                    value: "[[.Jms.KieServerJmsQueueResponse]]"
+                  - name: KIE_SERVER_JMS_QUEUE_EXECUTOR
+                    value: "[[.Jms.KieServerJmsQueueExecutor]]"
+                  - name: KIE_SERVER_JMS_ENABLE_SIGNAL
+                    value: "[[.Jms.KieServerJmsEnableSignal]]"
+                    # [[ if .Jms.KieServerJmsEnableSignal]]
+                  - name: KIE_SERVER_JMS_QUEUE_SIGNAL
+                    value: "[[.Jms.KieServerJmsQueueSignal]]"
+                    # [[end]]
+                  - name: KIE_SERVER_JMS_ENABLE_AUDIT
+                    value: "[[.Jms.KieServerJmsEnableAudit]]"
+                    # [[ if .Jms.KieServerJmsEnableAudit]]
+                  - name: KIE_SERVER_JMS_QUEUE_AUDIT
+                    value: "[[.Jms.KieServerJmsQueueAudit]]"
+                    # [[end]]
+                  - name: KIE_SERVER_JMS_AUDIT_TRANSACTED
+                    value: "[[.Jms.KieServerJmsAuditTransacted]]"
+                  - name: MQ_SERVICE_PREFIX_MAPPING
+                    value: "[[.KieName]]-amq7=AMQ"
+                  - name: AMQ_USERNAME
+                    value: "[[.Jms.KieServerJmsUsername]]"
+                  - name: AMQ_PASSWORD
+                    value: "[[.Jms.KieServerJmsPassword]]"
+                  - name: AMQ_PROTOCOL
+                    value: "tcp"
+                  - name: AMQ_QUEUES
+                    value: "[[.Jms.KieServerJmsAMQQueues]]"
+                  # JMS config END
+    ## KIE server deployment config END
+    ## AMQ deployment BEGIN
+    - metadata:
+        name: "[[.KieName]]-amq"
+        labels:
+          app: "[[$.ApplicationName]]"
+          application: "[[$.ApplicationName]]"
+          service: "[[.KieName]]-kieserver"
```

Comment:
This should probably just be:
`          service: "[[.KieName]]"`

like we see here in other config files:
https://github.com/kiegroup/kie-cloud-operator/blob/9547c5f534eb7a09e6dea1531cfa7f0f01bdfa74/config/common.yaml#L433

Reply:
unless you intended this to be:
`service: "[[.KieName]]-amq"` ??

Reply:
that's right, it should be .KieName